# Goal
float32 target_distance
---
# Result
bool success
string message
---
# Feedback
float32 current_distance

